examples/rest-api/README.md

```diff
@@ -0,0 +1,58 @@
+# Zync REST API example
+
+This example project implements Zync's REST API protocol to synchronize OAuth2 clients.
+
+## Prerequisites
+
+Given 3scale API is configured to use:
+  * OpenID Connect as the Authentication,
+  * "REST API" as a OpenID Connect Issuer Type and
+  * "http://id:secret@example.com/api" as OpenID Connect Issuer.
+
+When a 3scale application is created/updated/deleted zync will try to replay that change to "http://example.com/api".
```

Comment:
```suggestion
When you create, update or delete a 3scale application, Zync attempts to replay the change to "http://example.com/api".
```
_Zync_ is a product name, hence capitalized, unless the original name remains with small case.